ci/spelling-config.json

```diff
@@ -4,28 +4,19 @@
     "ignorePaths": [
         "**/.github/**",
         "**/ci/**"
-    ],    
+    ],
     "words": [
```

Comment:
Nit: We should add a comment explaining what goes in this list.

Reply:
yeah, that would be nice.  Though a bit outside the scope of this change, since json doesn't support comments in any straightforward manner (that I know of). I made a note in the lint issue (which is still in-progress) https://github.com/cncf/tag-security/issues/311#issuecomment-872676561 -- feel free to add your thoughts there.